### DISCUSSION TITLE
500 - interner server Fehler

### DISCUSSION THREAD

**Discussion by Nemo3006** [solectrus/solectrus#3671] [Q&A]
Hallo, seit Watchtower mir solectrus auf 18.0 aktualisiert hat bekomme ich den Fehler 500 - interner server Fehler. Es wurde nichts am raspberry pi oder an der hetzner cloud geändert. Der Fehler trat von ganz allein auf nach dem update. Was kann ich tun bzw. wie komme ich an die logs heran? Viele Grüße und schonmal danke!

**Comment by ledermann** [comment]
Klingt nach #3669, ist behoben mit v0.18.1 - bitte nochmal updaten.

**Reply by Nemo3006** [reply]
Danke. Hab ich übersehen. Für den Fall der Fälle: Kannst du mir schreiben wie ich an die logs komme?

**Reply by ledermann** [reply]
Klar. Es steht eigentlich auch in der Fehlermeldung: Einloggen per SSH und dann `docker compose logs`. 

Mehr dazu hier:
https://docs.solectrus.de/support/docker#protokoll-der-docker-container
